frontend/summary/summary/ExploreHeatmapPlot.js

```diff
@@ -419,35 +472,37 @@ class ExploreHeatmapPlot extends D3Visualization {
             .attr("y", d => this.y_scale(d.y_step) + this.y_scale.rangeBand() / 2);
 
         // enter/update
-        cells_data
+        this.cells_data
             .select(".exp_heatmap_cell_block")
             .transition()
             .style("fill", d => {
                 const value =
-                    tableDataFilters == null
+                    tableDataFilters.size == 0
                         ? d.rows.length
-                        : tableDataFilters.index === d.index
+                        : typeof _.find([...tableDataFilters], e => e.index === d.index) !==
```

Comment:
this is not very readable (and below). could this be modified or add a comment? maybe see `_.includes`